data/human/deep missions.txt

```diff
@@ -2431,3 +2431,320 @@ event "battle in arneb end"
+			choice
+				`	(Ask about the elves.)`
+					goto elves
+				`	(Ask about lost treasure.)`
+					goto treasure
+			label elves
+			`	The idea of Elves, or Alfar as they call them, intrigues you and you ask what they know of them.`
+			`	"The Alfar," Thorleif begins, "are said to have been a great help to us in the early days. Some say they spared us from plagues and protected us from monsters. Many still offer sacrifices and prayers to them, so that they remember us and know that we are friends and allies still."`
+			`	Thorleif shrugs, "This may be true, and good at that, but others think the Alfar, for all their apparent good works, cannot be trusted. Their minds are unknowable and filled with mischief, and their aims are not ours. This is the belief that our family holds to. They should be placated, but also feared. We must look to each other for support and strength, and trust not in the deeds of strangers."`
+					goto family
+			label treasure
+			`	You could always use a little extra money, and this trip, while engaging, has certainly not made you any richer, so you ask about hidden treasures. Helm and Thorleif both grin at this and glance at each other knowingly. They launch into tales of the riches of lost civilizations, or forgotten pirate hoards, or even early human kingdoms. It's easy to lose things on a fog-covered ocean world with floating islands, all too easy.`
+			`	When they are done regaling you with tales they mention to you that maybe some other time they'll take you out into the deep ocean and perhaps you'll find what you're looking for. They smile wolfishly at that and you think they might be putting you on. "Have you ever found treasure out here?" you ask.`
+			`	Thorleif shrugs, "coming here.. from Alfheim long ago, our descendants felt that the world itself was the real treasure. We have no need to go tracing dreams in the abyss."`
+			label family
+			`	As the conversation winds down and the hour grows late Helm excuses himself to bed, for he, like the rest of you, is bone tired after the day's events. You stand too, feeling your own need for sleep.`
+			`	"<first>," Thorleif says without looking up from the fire. "I've been thinking. I can never forgive him for what he did. Leaving me and mom like that. But I can, I think, make peace with it. I can accept the stone." He looks up and smiles, "You brought some luck with you, and a man should never argue with luck."`
+			`	He stands up, "We'll talk in the morning."`
+			choice
+				`	(You head to bed for some well deserved sleep.)`
+			`	The day is bright and the sky is clear. The storm long forgotten as you prepare your things. Out in the main room Thorleif and Helm are waiting for you, and Thorleif opens up his hand to accept the stone, which you gladly place into it. He smiles and looks it over, tracing his finger along its chiseled runes.`
```

Comment:
```suggestion
			`	The day is bright and the sky is clear, the storm long forgotten as you prepare your things. Out in the main room, Thorleif and Helm are waiting for you, and Thorleif opens up his hand to accept the stone, which you gladly place into it. He smiles and looks it over, tracing his finger along its chiseled runes.`
```
The second sentence is incomplete, so we'll just throw it into the first sentence to resolve it. I think the flow of the middle part could be improved, so here's my suggestion:
```suggestion
			`	The day is bright and the sky is clear, the storm long forgotten as you prepare your things. Both men meet you in the main room. Thorleif opens up his hand to accept the stone, which you gladly place into it. He smiles and looks it over, tracing his finger along its chiseled runes.`
```